portfolio/src/main/java/com/google/sps/data/ScheduleRequest.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.Collection;
+
+/**
+ * Models request that are sent to the algorithm. It can be used for both creation and import flow.
+ */
+public class ScheduleRequest {
+  private Collection<CalendarEvent> events;
+  private Collection<Task> tasks;
+  private final Instant workHoursStartTime;
+  private final Instant workHoursEndTime;
+
+  /**
+   * Constructs a request to be scheduled.
+   *
+   * @param events: events happening the day of the scheduling,
+   * @param tasks: the tasks a user would like to schedule on that day
+   * @param workHoursStartTimeString: a string representation of the work hours start time. The
+   *     string is in format YYYY-MM-DDTHH:MM:SSZ
+   * @param workHoursEndTimeString: a string representation of the work hours end time. The string's
+   *     format is the same as startTimeString's. All of these fields are required for a request
+   *     however, events and tasks can be. empty in the case where there are no events or no tasks.
+   *     For example an empty schedule for that day or the user hits start scheduling before they
+   *     add any tasks (maybe by accident).
+   */
+  public ScheduleRequest(
+      Collection<CalendarEvent> events,
+      Collection<Task> tasks,
+      String workHoursStartTimeString,
```

Comment:
Is it possible for this method to take in Instant for start time and end time?

Reply:
Since the servlet handles all of the unpacking at this point, I just removed this file.